pkg/grpc/api_test.go

```diff
@@ -645,14 +657,15 @@ func TestGetSecret(t *testing.T) {
 		},
 		"store3": {
 			DefaultAccess:  config.AllowAccess,
-			AllowedSecrets: []string{"good-key"},
+			AllowedSecrets: []string{"error-key", "good-key"},
 		},
 	}
 	expectedResponse := "life is good"
 	storeName := "store1"
 	deniedStoreName := "store2"
 	restrictedStore := "store3"
 	unrestrictedStore := "store4" // No configuration defined for the store
+	nonExistingStore := "store5"  // Non-existing store
```

Comment:
```suggestion
	nonExistingStore := "nonexistent"  // Non-existing store
```
These already have descriptive names, but I would also use descriptive values for them. Sometimes only the values appears in the log and could be helpful when debugging as well.